airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,175 @@
+      record_selector:
+        $ref: "#/definitions/record_selector"
+        record_filter:
+          condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Issues https://docs.sentry.io/api/events/list-a-projects-issues/
+  issues:
+    type: DeclarativeStream
     $parameters:
       name: "issues"
-    primary_key: "id"
-    schema_loader:
-      $ref: "#/definitions/schema_loader"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/issues/"
+      cursor_field: "lastSeen"
     retriever:
-      $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/issues/"
         request_parameters:
           statsPeriod: ""
-          query: ""
+          query: "lastSeen:>{{ stream_state.get(parameters['cursor_field']) or '1900-01-01T00:00:00.0Z' if stream_state else '1900-01-01T00:00:00.0Z' }}"
+      record_selector:
+        $ref: "#/definitions/record_selector"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Projects https://docs.sentry.io/api/projects/list-your-projects/
+  projects:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream_incremental"
     $parameters:
       name: "projects"
-    primary_key: "id"
-    schema_loader:
-      $ref: "#/definitions/schema_loader"
-    retriever:
-      $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
-      requester:
-        $ref: "#/definitions/requester"
-        path: "projects/"
-      paginator:
-        $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+      primary_key: "id"
+      path: "projects/"
+      cursor_field: "dateCreated"
+
+  # Stream Project Detail https://docs.sentry.io/api/projects/retrieve-a-project/
+  project_detail:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream_full_refresh"
     $parameters:
       name: "project_detail"
-    primary_key: "id"
-    schema_loader:
-      $ref: "#/definitions/schema_loader"
-    retriever:
-      $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
-      requester:
-        $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/"
-      paginator:
-        type: NoPagination
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/"
+
+  # Stream Releases https://docs.sentry.io/api/projects/retrieve-a-project/
+  releases:
```

Comment:
`releases` was not fully incremental but only semi-incremental. This is not supported in the current low-code CDK hence I don't think this would work as the API does not allow for filtering using query params.

Reply:
`base_stream_incremental` is implemented with records filtration [here](https://github.com/airbytehq/airbyte/pull/35755/files#diff-7962b9b9cf6a7a29a9010fcc358e43a160fa424c358e037ed39779594ec0d42aR47) , so all records will be filtered and returned as like in python version. Here is doc with comparison: [link](https://docs.google.com/spreadsheets/d/1TF-nT91DP1qcQ1Mko2e2A2gwv05BgA0CU8DGdQblbdE/edit#gid=0) .

Reply:
Oh, I didn't see that, my bad! @girarda it goes a bit against the "don't pass stream_state for interpolation" thing we were discussing about. TLDR @darynaishchenko is that stream_state can change at any point and it is not so clear what is the value at a specific point in time. I think for now this is fine, but it probably bumps up the priority of "semi-incremental" in low-code

Reply:
Agreed, Maxime. Let's try to slot that in for Q1B